example/src/App.js

```diff
@@ -0,0 +1,75 @@
+import { useRouting, Link } from '@kaliber/routing'
+
+export default function App({ initialLocation }) {
+  const { pick } = useRouting({ initialLocation })
+  return (
+    <>
+      <Navigation />
+      {pick(
```

Comment:
I feel `pick` might be a confusing name for this function (at least in a public API) since the name `pick` is commonly used for creating an object composed of picked object properties. 

https://www.npmjs.com/package/object.pick
https://lodash.com/docs/4.17.15#pick

Reply:
Would `match` be better?

Reply:
Or `routes`?

Reply:
I like `match` or perhaps `matchRoute`?